docker/ogl_obr.dockerfile

```diff
@@ -0,0 +1,58 @@
+# Container image that runs your code    
+FROM ghcr.io/foamscience/of-unit-testing-paper:base-2206    
+SHELL ["/bin/bash", "-c"]    
+# We need to set this enviroment variable    
+# since different OpenFOAM versions have    
+# a different name for this case    
+ENV CYCLIC_CASE=periodicPlaneChannel    
+# Set the PATH variable for local pip installs    
+ENV PATH="${PATH}:${HOME}/.local/bin"    
+                                                                                                                                                                    
+# Set the default solver execution template                                                                                                                         
+# we set this to explicitely allow oversubscribe                                                                                                                    
+ENV OBR_RUN_CMD="mpirun --oversubscribe --bind-to core --map-by core -np {np} {solver} -parallel -case {path}/case >  {path}/case/{solver}_{timestamp}.log 2>&1"    
+                                                                                                                                                                    
+RUN sudo apt update && sudo apt install -y pip ninja-build    
+                                                              
+# update python environment                    
+RUN python3.10 -m pip install pip --upgrade            
+                                                       
+# Clone dependencies                                   
+RUN cd $HOME && \                                      
+    git clone https://github.com/hpsim/OGL.git && \    
+    git clone https://github.com/hpsim/OBR.git && \    
+    pip install ./OBR                                  
+                                                           
```

Comment:
Maybe do not include anything from this point on in the image.
The rest can be mentioned in the README as a three-steps introduction to what commands reviewers are expected to run

Reply:
The image is supposed to provide the software and the environment, reviewers can do whatever they want.

Reply:
well the actual cutting line should be after all the cmakes and inits; so maybe line `RUN obr run -o runParallelSolver`?

Reply:
The reason is that the `RUN obr run -o runParallelSolver` takes very long, for two reasons:
- The runs are usually done concurrently, on different runners, here they are run in serial.
- A debug flag is set, which lets the application wait some seconds for a debugger to attach.

I'll include the `runParallelSolver` and mention that there is the option to rerun the test cases.